lib/integrations/awsoidc/eks_enroll_clusters.go

```diff
@@ -278,7 +277,10 @@ func EnrollEKSClusters(ctx context.Context, log logrus.FieldLogger, clock clockw
 		group.Go(func() error {
 			resourceId, err := enrollEKSCluster(ctx, log, clock, credsProvider, clt, proxyAddr, eksClusterName, req)
 			if err != nil {
-				log.WithError(err).Debugf("failed to enroll EKS cluster %q", eksClusterName)
+				log.WarnContext(ctx, "Failed to enroll EKS cluster",
```

Comment:
Was the log level increased on purpose?

```suggestion
				log.DebugContext(ctx, "Failed to enroll EKS cluster",
```

Reply:
It was not, thank you for noticing it.